gensim/models/ldaseqmodel.py

```diff
@@ -0,0 +1,1100 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+# Based on Copyright (C) 2016 Radim Rehurek <radimrehurek@seznam.cz>
+
+
+"""
+
+Inspired by the Blei's original DTM code and paper.
+Original DTM C/C++ code: https://github.com/blei-lab/dtm
+DTM Paper: https://www.cs.princeton.edu/~blei/papers/BleiLafferty2006a.pdf
+
+
+TODO:
+The next steps to take this forward would be:
+
+    1) Include DIM mode. Most of the infrastructure for this is in place.
+    2) See if LdaPost can be replaces by LdaModel completely without breakign anything.
+    3) Heavy lifting going on in the sslm class - efforts can be made to cythonise mathematical methods.
```

Comment:
how much time is spent there according to line profiler? is it really a perf bottleneck?